src/encoded/audit/experiment.py

```diff
@@ -4018,6 +4018,24 @@ def audit_experiment_no_processed_data(value, system, files_structure):
         yield AuditFailure('lacking processed data', detail, level='WARNING')
 
 
+def audit_experiment_inconsistent_analyses_files(value, system, files_structure):
+    processed_data = files_structure.get('processed_data')
+    if processed_data and 'analyses' in value:
+        for file_object in processed_data:
+            for analysis in 'analyses':
+                if file_object in analysis:
+                    return
+                if file_object not in analysis:
+                    detail = ('Experiment {} '
+                        'contains a processed file {} '
+                        'that is not in an analysis'.format(
+                            audit_link(path_to_text(value['@id']), value['@id']),
+                            audit_link(path_to_text(file_object['@id']), file_object['@id']),
+                        )
+                    )
+                    yield AuditFailure('inconsistent analyses files', detail, level='INTERNAL_ACTION')
```

Comment:
I think this code is incorrect. Should have been something like:
`
for analysis in value['analyses']:
...
`
and I think that it should not be file_object in analysis, but file_object['@id'] in analysis